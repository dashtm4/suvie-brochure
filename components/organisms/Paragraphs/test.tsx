import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Paragraphs } from '.'

describe('<Paragraphs />', () => {
  const children = 'Test children 1 \n Test children 2'

  it('should renders correct <Paragraphs />', () => {
    const wrapper = shallow(<Paragraphs>{children}</Paragraphs>)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
