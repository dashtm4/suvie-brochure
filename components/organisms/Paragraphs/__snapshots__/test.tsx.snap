// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraphs /> should renders correct <Paragraphs /> 1`] = `
<Fragment>
  <p
    className="mb-2 last:mb-0"
    key="0"
  >
    Test children 1 
  </p>
  <p
    className="mb-2 last:mb-0"
    key="1"
  >
     Test children 2
  </p>
</Fragment>
`;
